Packs/Core/Playbooks/playbook-External_Login_Password_Spray.yml

```diff
@@ -0,0 +1,1474 @@
+    id: "59"
+    taskid: df38478e-ae21-469c-8454-bd65afc04aeb
+    type: regular
+    task:
+      id: df38478e-ae21-469c-8454-bd65afc04aeb
+      version: -1
+      name: Prepare list of usernames for soft remediation
+      description: Saves a general list of all the users that will require soft remediation. Specific types of remediation may need the names to appear in different structures or syntax.
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: UsersToDisconnect
+      value:
+        complex:
+          root: SuccessfulLoginUsernames
+          transformers:
+          - operator: replace
+            args:
+              limit: {}
+              replaceWith:
+                value:
+                  simple: \
+              toReplace:
+                value:
+                  simple: \\
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: ''''
+              suffix:
+                value:
+                  simple: ''''
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ','
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1150,
+          "y": 1360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "63":
+    id: "63"
+    taskid: 34b3eb95-9bb9-46cf-8bae-dcc640a0aee2
+    type: condition
+    task:
+      id: 34b3eb95-9bb9-46cf-8bae-dcc640a0aee2
+      version: -1
+      name: What OS is running on the host?
+      description: Checks if the operating system reported by the agent of the machine to which the users logged in is Windows or another OS.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "78"
+      Windows:
+      - "65"
+    separatecontext: false
+    conditions:
+    - label: Windows
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: alert
+                accessor: hostos
+                transformers:
+                - operator: uniq
+                - operator: FirstArrayElement
+            iscontext: true
+          right:
+            value:
+              simple: AGENT_OS_WINDOWS
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 2265
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "65":
+    id: "65"
```

Comment:
Why are we not setting the password for users who connected via RDP in soft remediation?